ntbs-service/Migrations/NtbsContextModelSnapshot.cs

```diff
@@ -12574,7 +12576,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsUnique()
                         .HasFilter("[ServiceAdGroup] IS NOT NULL");
 
-                    b.ToTable("TBService");
+                    b.ToTable("TbService");

```

Comment:
Is the change of case here going to be problematic?

Reply:
Think this came about as a rename that I did. I didn't consider that we hadn't manually specified a table name like we have in a lot of other instances.
I'm not sure whether it would be problematic - but I imagine if it would be we would have experienced it by now. By default SQL doesn't care about casing... but I don't know if `Sql to Entities` might be doing some exotic things where casing would matter.